test/helpers/kubectl.go

```diff
@@ -2213,7 +2212,7 @@ func (kub *Kubectl) ExecInHostNetNS(ctx context.Context, node, cmd string) (resu
 	// which runs in the host netns. Also, the log-gathering pods lack some
 	// packages, e.g. iproute2.
 	selector := fmt.Sprintf("%s --field-selector spec.nodeName=%s",
-		LogGathererSelector, node)
+		logGathererSelector(true), node)
```

Comment:
In the end this is `k8s-app=cilium-test-logs` and all others are `LogGathererSelector` for which `LogGathererSelector = logGathererSelector(false)`

Reply:
I introduced the helper function to prevent from accidentally selecting all log-gatherers when a user means only those log-gatherers which run on cilium nodes.